node/e2e/client_service_binary.test.ts

```diff
@@ -0,0 +1,217 @@
+import {ClientServiceWithBinary} from '../client_service_binary';
```

Comment:
It might be a stupid question, but what is the difference in the purpose between `client_service_binary.test` and `integration_test_binary_request`? I'm wondering if we can refactor them a little. At least the naming scheme can be unified for the better understanding if they are both integration tests or e2e tests.

Reply:
> It might be a stupid question, but what is the difference in the purpose between `client_service_binary.test` and `integration_test_binary_request`? I'm wondering if we can refactor them a little. At least the naming scheme can be unified for the better understanding if they are both integration tests or e2e tests.

@jnmt 
Jun, Thank you for the question.

The purpose of `client_service_binary.test.ts` and `integration_test_binary_request` are the same.

Sorry, the naming must have confused you.

The difference is I use Jest for testing because it has complete features (mocking, assertion, ... etc)

Reply:
Ah! Sorry I think I made a mistake with copying `integration_test_binary_request` to this PR. We don't need it. I copied it from the old Node repo locally for a reference.

I deleted it in https://github.com/scalar-labs/scalardl-javascript-client-sdk/pull/29/commits/cf53470ebedd4c8a7cfc16fbba80e89fad467d96

Reply:
Got it! Thanks!